test/fail_compilation/fail13292.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+errmsg="$(! $DMD -c -o- -main -m32 -m64 ${EXTRA_FILES}/minimal/object.d 2>&1 > /dev/null)"
+expected="Error: Conflicting target architectures specified: -m32 and -m64."
+
+if [ "$errmsg" != "$expected" ]; then exit 1; fi
+
+exit 0
```

Comment:
Avoid bash tests, either convert this into a dshell test or create `fail_compilation/fail13292.d` with an appropriate `TEST_OUTPUT section`, e.g.
```
/*
REQUIRED_ARGS: -m32 -m64
TEST_OUTPUT:
----
Error: Conflicting target architectures specified: $?:32mscoff=-m32mscoff and -m32|32=-m32 and -m64|64=-m64 and -m32$.
----
*/
```